### DISCUSSION TITLE
Is Voyager not compatible with Kotlin/Wasm ? Unable to run the app on browser.

### DISCUSSION THREAD

**Discussion by DeepakRattan** [adrielcafe/voyager#451] [Q&A] [answered]
I am trying to use Voyager for Android,IOS, Desktop and Web for a compose multiplatform project. It is working fine for Android, iOS and Desktop but for browser I am getting error w.r.t. kotlin/wasm. Please let me know if we are having support for browser?

**Comment by Syer10** [comment, marked as answer]
You need to use 1.1.0-beta2 since there were issues with the latest compose + release voyager on Wasm.
https://github.com/adrielcafe/voyager/issues/315
